pilot/pkg/networking/v1alpha3/listener.go

```diff
@@ -23,7 +23,7 @@ import (
 	"strings"
 	"time"
 
-	xdsapi "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
```

Comment:
Please keep it xdsapi, we're using it consistently and it's much easier to read than 'v2'